frontend/public/components/events.jsx

```diff
@@ -227,7 +227,10 @@ export class EventsList extends React.Component {
         <Helmet>
           <title>Events</title>
         </Helmet>
```

Comment:
I think this should move to `EventStreamPage` component.

Reply:
Yes. Moved as suggested